src/js/popper.js

```diff
@@ -0,0 +1,1210 @@
+    }
+    return null;
+  }
+  function H() {
+    return (
+      (this.state.isDestroyed = !0),
+      W(this.modifiers, "applyStyle") &&
+        (this.popper.removeAttribute("x-placement"),
+        (this.popper.style.position = ""),
+        (this.popper.style.top = ""),
+        (this.popper.style.left = ""),
+        (this.popper.style.right = ""),
+        (this.popper.style.bottom = ""),
+        (this.popper.style.willChange = ""),
+        (this.popper.style[B("transform")] = "")),
+      this.disableEventListeners(),
+      this.options.removeOnDestroy &&
+        this.popper.parentNode.removeChild(this.popper),
+      this
+    );
+  }
+  function A(e) {
+    var t = e.ownerDocument;
+    return t ? t.defaultView : window;
+  }
+  function M(e, t, o, i) {
+    var r = "BODY" === e.nodeName,
+      p = r ? e.ownerDocument.defaultView : e;
+    p.addEventListener(t, o, { passive: !0 }),
+      r || M(n(p.parentNode), t, o, i),
+      i.push(p);
+  }
+  function F(e, t, o, i) {
+    (o.updateBound = i),
+      A(e).addEventListener("resize", o.updateBound, { passive: !0 });
+    var r = n(e);
+    return (
+      M(r, "scroll", o.updateBound, o.scrollParents),
+      (o.scrollElement = r),
+      (o.eventsEnabled = !0),
+      o
+    );
+  }
+  function I() {
+    this.state.eventsEnabled ||
+      (this.state = F(
+        this.reference,
+        this.options,
+        this.state,
+        this.scheduleUpdate
+      ));
+  }
+  function R(e, t) {
+    return (
+      A(e).removeEventListener("resize", t.updateBound),
+      t.scrollParents.forEach(function (e) {
+        e.removeEventListener("scroll", t.updateBound);
+      }),
+      (t.updateBound = null),
+      (t.scrollParents = []),
+      (t.scrollElement = null),
+      (t.eventsEnabled = !1),
+      t
+    );
+  }
+  function U() {
+    this.state.eventsEnabled &&
+      (cancelAnimationFrame(this.scheduleUpdate),
+      (this.state = R(this.reference, this.state)));
+  }
+  function Y(e) {
+    return "" !== e && !isNaN(parseFloat(e)) && isFinite(e);
+  }
+  function V(e, t) {
+    Object.keys(t).forEach(function (o) {
+      var n = "";
+      -1 !== ["width", "height", "top", "right", "bottom", "left"].indexOf(o) &&
+        Y(t[o]) &&
+        (n = "px"),
+        (e.style[o] = t[o] + n);
+    });
+  }
+  function j(e, t) {
+    Object.keys(t).forEach(function (o) {
+      var n = t[o];
+      !1 === n ? e.removeAttribute(o) : e.setAttribute(o, t[o]);
+    });
+  }
+  function q(e, t) {
+    var o = e.offsets,
+      n = o.popper,
+      i = o.reference,
+      r = $,
+      p = function (e) {
+        return e;
+      },
+      s = r(i.width),
+      d = r(n.width),
+      a = -1 !== ["left", "right"].indexOf(e.placement),
+      l = -1 !== e.placement.indexOf("-"),
+      f = t ? (a || l || s % 2 == d % 2 ? r : Z) : p,
+      m = t ? r : p;
+    return {
+      left: f(1 == s % 2 && 1 == d % 2 && !l && t ? n.left - 1 : n.left),
```

Comment:
Codacy found an issue: [Use ===/!== to compare with true/false or Numbers](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)